backend/migrations/versions/698e5724baae_last_modified_column.py

```diff
@@ -0,0 +1,32 @@
+"""last_modified_column
+
+Revision ID: 698e5724baae
+Revises: 8b5132609f1f
+Create Date: 2023-11-08 20:26:08.758322
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '698e5724baae'
+down_revision = '0ea2257f1dc6'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('users', schema=None) as batch_op:
+        batch_op.add_column(sa.Column('last_modified', sa.DateTime(), nullable=True))
```

Comment:
so one thing i noticed is when you run the initial invite user script it doesn't work because this doesn't actually have a default - I believe if you add these:
```
server_default=text('CURRENT_TIMESTAMP'), onupdate=text('CURRENT_TIMESTAMP')
```

it should solve this issue.

Also, make sure to set this as `nullable=False`